torch/_refs/nn/functional/__init__.py

```diff
@@ -1201,7 +1201,13 @@ def pairwise_distance(
     eps: NumberType = 1e-6,
     keepdim=False,
 ) -> TensorLikeType:
-    return torch.linalg.vector_norm(x1 - x2 + eps, ord=p, dim=-1, keepdim=keepdim)
+    if isinstance(p, bool):
```

Comment:
Do we actually want to support p as a bool and complex or is the typing of p just incorrect and should be narrow. Supporting Boolean and complex here feels weird.